app/views/projects/dataset/keys/_numeric_key.html.erb

```diff
@@ -1,2 +1,4 @@
-<%= render partial: 'projects/dataset/keys/base_key',
-           locals: { f: f, key: key } %>
+<div class="dataset-key">
+  <%= f.label :value, key.title %>
+  <%= f.text_field :value, rows: 5 %>
```

Comment:
`rows` is only for `textarea` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input